Packs/DarktraceEventCollector/ModelingRules/DarktraceEventCollector/DarktraceEventCollector.xif

```diff
@@ -0,0 +1,43 @@
+[MODEL: dataset ="darktrace_darktrace_raw"]
+alter
+    xdm.alert.description = json_extract_scalar(model, "$.then.description"),
+    xdm.event.description = json_extract_scalar(model, "$.then.description"),
+    xdm.alert.mitre_tactics = arraymap(json_extract_array(model, "$.then.mitre.tactics"), trim("@element", "\"")),
+    xdm.alert.mitre_techniques = arraymap(json_extract_array(model, "$.then.mitre.techniques"), trim("@element", "\"")),
+    xdm.alert.original_alert_id = to_string(pbid),
+    xdm.event.id = to_string(pbid),
+    xdm.alert.original_threat_id = to_string(json_extract_scalar(model, "$.then.uuid")),
+    xdm.alert.original_threat_name = json_extract_scalar(model, "$.then.name"),
+    xdm.event.type = json_extract_scalar(model, "$.then.name"),
+    xdm.alert.severity = to_string(score),
+    priority = json_extract_scalar(model, "$.then.priority"),
+    xdm.event.tags = arraymap(json_extract_array(model, "$.then.tags"), trim("@element", "\"")),
+    xdm.network.rule = to_string(json_extract_scalar(model, "$.then.pid")),
+    xdm.observer.action = json_extract_scalar(model, "$.then.actions.antigena.action"),
+    xdm.observer.name = json_extract_scalar(device, "$.hostname"),
```

Comment:
should have been removed

Reply:
The request was to remove the coalesce between the hostname and the ip as this is already displayed in another field.